Add config model repo test controller

myoungkeun-oh opened the pull request.

myoungkeun-oh committed:
test after merge

myoungkeun-oh committed:
분석 & 주석 추가

주석 추가 => SecurityConfig, PrincipalDetails, PrincipalDetailsService, UserRepository

OMG-maker commented:
주석 추가 => SecurityConfig, PrincipalDetails, PrincipalDetailsService, UserRepository

myoungkeun-oh committed:
security login Test

PostMan 에서 post 방식으로 /loginProc 호출 실험
이후 성공한다면 username 아닌 userEmail 로 로그인 테스트 진행

OMG-maker commented:
o
